File: python/1221_分割平衡字符串_easy.py
class Solution:
    def balancedStringSplit(self, s: str) -> int:
        ans = 0
        i, j = 0, 0
        for ch in s:
            if ch == 'R':
                i += 1
            else:
                j += 1
            if i == j:
                i, j = 0, 0
                ans += 1
        return ans

File: cpp/1221_分割平衡字符串_easy.cpp
class Solution {
public:
    int balancedStringSplit(string s) {
        int ans = 0, i = 0, j = 0;
        for (auto ch : s) {
            if (ch == 'R')
                i++;
            else
                j++;
            if (i == j) {
                i = 0;
                j = 0;
                ans++;
            }
        }
        return ans;
    }
};